cobalt/script/value_handle.h

```diff
@@ -38,16 +39,48 @@ class ValueHandle {
 
 typedef ScriptValue<ValueHandle> ValueHandleHolder;
 
-struct BufferDeleter {
-  void operator()(uint8_t* buffer) { SbMemoryDeallocate(buffer); }
-};
-using DataBufferPtr = std::unique_ptr<uint8_t[], BufferDeleter>;
 
-struct DataBuffer {
-  DataBufferPtr ptr;
-  size_t size;
+class StructuredClone : public v8::ValueSerializer::Delegate,
+                        public v8::ValueDeserializer::Delegate {
+ public:
+  explicit StructuredClone(const ValueHandleHolder& value);
```

Comment:
Maybe add a comment explaining that the value passed into the constructor gets serialized immediately.